packages/unified-lint-rule/index.d.ts

```diff
@@ -18,15 +18,12 @@ export interface RuleMeta {
 export function lintRule<Tree extends Node = Node, Options = unknown>(
   name: string | RuleMeta,
   rule: Rule<Tree, Options>
-): Plugin<
-  void[] | [Options | Label | Severity] | [boolean | Label | Severity, Options],
-  Tree
->
+): Plugin<void[] | [Options | Severity] | [boolean | Severity, Options], Tree>
```

Comment:
Doing so makes the `lintRule()` signature (which *is* external) a little simpler, I think?